src/components/GroupByRow.tsx

```diff
@@ -41,10 +41,11 @@ type Props = {
   readonly isAdAnalytics: boolean;
   readonly onChange: (newGroupBys: Array<QueryAttribute | QueryAdAttribute>) => void;
   readonly groupBys: Array<QueryAttribute | QueryAdAttribute>;
+  readonly queryEditorId: string;
 };
 
 export function GroupByRow(props: Props) {
-  const paddingTop= props.groupBys.length === 0 ? 4 : 0
+  const paddingTop = props.groupBys.length === 0 ? 4 : 0;
```

Comment:
if formatting is wrong, I would create new PR based on `main` and solve it there:
- format project code
- implement "format check" in CI
- merge that PR to `main` and then merge latest `main` here, to reduce code change lines in this PR, because they are not relevant for playwright task

Reply:
Addressed in this PR: https://github.com/bitmovin/analytics-grafana-datasource/pull/94